front/pages/api/w/[wId]/vaults/[vId]/data_sources/[name]/managed.ts

```diff
@@ -0,0 +1,138 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
```

Comment:
Added 2 routes to be able to delete a datasources (static/managed).
We'll need to do it also for the patch.